submissions/EmmanuelUdoh/HTML/landing.css

```diff
@@ -0,0 +1,208 @@
+body{
+    box-sizing: border-box;
+}
+#wrapper{font-family: Verdana, Geneva, Tahoma, sans-serif;}
+
+.container{width: 90%;
```

Comment:
indentation 🙄

Reply:
Generally review the CSS section on the style guide and make necessary corrections